Documentation/ceph-dashboard.md

```diff
@@ -59,6 +59,26 @@ To get the dashboard to use hyperlinks that include your prefix, you can set the
       urlPrefix: /ceph-dashboard
 ```
 
+The port that the dashboard is served on may be changed from the defaults using
+the `port` settings. The corresponding K8s service exposing the port will
+automatically be updated:
+```yaml
+  spec:
+    dashboard:
+      port: 43219
```

Comment:
How about using 8443 as the default port in the example? This will make it clear what our default is as well. Not too many people will be using luminous going forward so https seems like the good default.

Reply:
could you add a bullet to the PendingReleaseNotes.md to mention the new dashboard settings.